<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>551</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <attribute name="title">
     <string>Adquisición de fotos</string>
    </attribute>
    <widget class="QPushButton" name="take_picture">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>440</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(10, 255, 255, 130);
font: 75 9pt &quot;Segoe Print&quot;;
</string>
     </property>
     <property name="text">
      <string>Take picture</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>511</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Picture</string>
     </property>
     <widget class="QLabel" name="img_cont">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>401</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>TIC1/6Sobre capacidad.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Name_Archiv</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 30, 255, 130);
font: 75 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>Save name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="fotor">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>440</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(10, 255, 255, 130);
font: 75 9pt &quot;Segoe Print&quot;;
</string>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Adquisición de videos</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>511</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Video</string>
     </property>
     <widget class="QLabel" name="img_cont_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>421</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>TIC1/6Sobre capacidad.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Control_video</string>
     </property>
     <widget class="QPushButton" name="a">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(10, 255, 255, 130);
font: 75 9pt &quot;Segoe Print&quot;;
</string>
      </property>
      <property name="text">
       <string>Take video</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Name_Archiv</string>
     </property>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 30, 255, 130);
font: 75 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>Save name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>take_picture</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>take_video()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>-34</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>take_foto()</slot>
  <slot>take_video()</slot>
 </slots>
</ui>
